packages/core/src/common/command.ts

```diff
@@ -297,8 +301,12 @@ export class CommandRegistry implements CommandService {
         const handlers = this._handlers[commandId];
         if (handlers) {
             for (const handler of handlers) {
-                if (!handler.isVisible || handler.isVisible(...args)) {
-                    return handler;
+                try {
+                    if (!handler.isVisible || handler.isVisible(...args)) {
+                        return handler;
+                    }
+                } catch (error) {
+                    this.logger.error(error);
```

Comment:
Should we instead use `console` instead? https://github.com/theia-ide/theia/issues/1675#issuecomment-450819568

Reply:
We should add new section `Logging` to the coding guidelines and write it down. @vince-fugnitto Could you do it?